CharacterCreation/Class-Specific Documentation/Hunters_Dog.txt

```diff
@@ -0,0 +1,57 @@
+==============================
+Hunter's Dog Rules and Stats
+==============================
+
+
+===== Dog Rules =====
+
+    The Dog cannot use weapons, armor nor non-food items. 
+
+    The Dog is smaller than most combatants and moves very quickly. If the Dog 
+moved at least 8m on their last turn, they are considered to be in partial cover
+from all angles. 
+
+    The Dog has no guard in melee. Instead, up to four times per round the dog 
+may roll a reflex save of DC 70 to dodge a melee attack.
+
+===== Dog Stats =====
+
+Health:     80
+Strength:   4
+Perception: 12
+Fortitude:  3
+Charisma:   8
+Intelligence: 2
+Dexterity:  12
+Luck:       5
+Will:       10
+Reflex:     36
+Shock:      -2
+Move Speed Per Action: 6
+Death Saving Throws: 3
+Nanites:    0
+
+    The Dog has a +(level x 4) bonus to investigate and awareness checks that 
+they can use their sense of smell to help with. 
+
+    The Dog can use tricks like "lay down," "shake," "fetch," or "chase your 
+tail" to charm or distract characters in the area. When they do, the target must
+succeed on a Will save opposed by the dog's Charisma (+12) performance roll.
```

Comment:
Performance is a skill, so we may want to either say "the dog's Charisma (+12) roll" or use standard skill notation- "the dog's Performance (Cha) roll".